mail/mail-policy.go

```diff
@@ -318,3 +318,48 @@ func SendMailRenewDraft(policy models.Policy, from, to, cc Address, flowName str
 		IsApp:       true,
 	})
 }
+
+func SendMailRenewDraftV2(policy models.Policy, from, to, cc Address, flowName string, hasMandate bool) error {
+	var (
+		bodyData BodyData
+		err      error
+	)
+
+	bodyData = getPolicyRenewDraftBodyData(policy, hasMandate)
+
+	templateFile, err := lib.GetFilesByEnvV2(fmt.Sprintf("mail/%s/%s.html", flowName, renewDraftTemplateType))
+	if err != nil {
+		return err
+	}
+
+	messageBody, err := fillTemplateV2(templateFile, &bodyData)
+	if err != nil {
+		return err
+	}
+
+	title := policy.NameDesc
+	subtitle := fmt.Sprintf("La tua polizza n° %s si rinnova il %s, provvedi al pagamento.", policy.CodeCompany,
+		bodyData.RenewDate)
+	if hasMandate {
+		subtitle = fmt.Sprintf("La tua polizza n° %s si rinnova il %s, pagamento senza pensieri.",
+			policy.CodeCompany, bodyData.RenewDate)
+	}
+	subject := fmt.Sprintf("%s - %s", title, subtitle)
+
+	err = SendMailV2(MailRequest{
+		FromAddress: from,
+		To:          []string{to.Address},
+		Cc:          cc.Address,
+		Message:     messageBody,
+		Title:       title,
+		SubTitle:    subtitle,
+		Subject:     subject,
+		IsHtml:      true,
+		IsApp:       true,
+	})
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
since we do not do extra work with the error, we could return directly the result from SendMailV2
`return SendMailV2(...)`